src/Winium.Desktop.Driver/CommandExecutors/QuitExecutor.cs

```diff
@@ -8,7 +15,26 @@ protected override string DoImpl()
         {
             if (!this.Automator.ActualCapabilities.DebugConnectToRunningApp)
             {
-                if (!this.Automator.Application.Close())
+                // If application had exited, find and terminate all children processes
+                if (this.Automator.Application.HasExited())
+                {
+                    List<Process> children = new List<Process>();
+                    children = this.Automator.Application.GetChildPrecesses(this.Automator.Application.GetProcessId());
+                    if (children.Count == 0)
+                    {
+                        children = this.Automator.Application.GetAllPrecessesByName(this.Automator.Application.GetProcessName());
```

Comment:
GetAllPrecessesByName --> is this sure that this is the sub/child process ? Let's discuss this.